support-frontend/assets/components/legal/termsPrivacy/termsPrivacy.jsx

```diff
@@ -100,37 +101,40 @@ function TermsPrivacy(props: PropTypes) {
   const sourceIsNotAppleNews = props.referrerSource !== 'APPLE_NEWS';
   const sourceIsNotGoogleAMP = props.referrerSource !== 'GOOGLE_AMP';
 
-  const shouldShowPhilanthropicAsk = (
-    isUSContributor &&
-    isNotOneOffContribution &&
-    sourceIsNotAppleNews &&
-    sourceIsNotGoogleAMP
-  );
+  const shouldShowPhilanthropicAsk =
+    isUSContributor && isNotOneOffContribution && sourceIsNotAppleNews && sourceIsNotGoogleAMP;
 
   return (
     <>
       <div className="component-terms-privacy">
-        {props.contributionType !== 'ONE_OFF' ?
+        {props.contributionType !== 'ONE_OFF' ? (
+          <div className="component-terms-privacy__change">
+            Monthly contributions are billed each month and annual contributions are billed once a year.{' '}
+            <strong>You can change how much you give or cancel your contributions at any time.</strong>
+          </div>
+        ) : null}
+        {props.paymentMethod === 'Sepa' ? (
           <div className="component-terms-privacy__change">
-            Monthly contributions are billed each month and annual contributions are billed once a year.
-            You can change how much you give or cancel your contributions at any time.
+            By providing your IBAN and confirming this payment, you are authorizing Guardian News & Media Ltd and
```

Comment:
Our styleguide is to use UK English, so we should change 'authorizing' to 'authorising'.